src/main/java/com/sobytylnik/quiz/QuizAnswers.java

```diff
@@ -0,0 +1,56 @@
+package com.sobytylnik.quiz;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class QuizAnswers {
+    private Quiz quiz;
+    private Map<Integer, List<Option>> answers;
+
+    public QuizAnswers(Quiz quiz) {
+        this.quiz = quiz;
+        this.answers = new HashMap<>();
+        for (Question elem:quiz.getQuestions()) {
+            answers.put(elem.getId(),null);
+        }
+    }
+
+    public void addAnswer(int questionID, List<Option> answers){
+        this.answers.put(questionID,answers);
+    }
+
+    public Map<Integer, List<Option>> getAnswers() {
+        return answers;
+    }
+
+    public Quiz getQuiz() {
+        return quiz;
+    }
+
+    public void setQuiz(Quiz quiz) {
+        this.quiz = quiz;
+    }
+
+    private String displayAnswers(int questionID){
```

Comment:
What is the difference between this method and `displayAnswer(int questionID)`?
Moreover, this method is private and never used.
Do we need it?